.circleci/config.yml

```diff
@@ -119,4 +186,10 @@ workflows:
               ignore: /.*/
             tags:
               only: /^production.*/
-
+          context: binary-frontend-artifact-upload  # using context to retrieve shared secret from circle ci
+      - deploy_production:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^production.*/
```

Comment:
deriv-app does releases based on PR being merged to dev/master branch. Please remove the tags and use branches instead.

Reply:
i got previous circle config based on tagging and also from readme I found that release happens on this repo based on tagging
![image](https://user-images.githubusercontent.com/59427711/77295074-3db6e980-6d20-11ea-9413-ac6bb4bc1229.png)